en/catalog.json

```
@@ -1248,6 +1248,18 @@
     "brokerLoading": {
       "preparingPayment": "Preparing your payment details"
     },
+    "catalogFilterIneligibilityReasons": {
+      "integrationNotSupportedCountry": "Unavailable in your region",
+      "integrationMultipleTokenNetworkCompatibility": "Unsupported token and network",
+      "integrationSingleTokenNetworkCompatibility": "Unsupported token and network",
+      "lessThanIntegrationMinAmount": "Amount below the minimum",
+      "platformNotSupported": "Unsupported on this device",
+      "moreThanIntegrationMaxAmount": "Amount above the maximum",
+      "sdkVersionNotSupported": "Unavailable",
+      "travelRuleAmountLimit": "Compliance eligibility restriction",
+      "travelRuleVaspIdRequired": "Compliance eligibility restriction",
+      "travelRuleWalletOwnershipRequired": "Compliance eligibility restriction"
+    },
```

Suggest put them into the catalogPage and remove duplicates:

```suggestion
"catalogPage": {
        "integrationIneligibilityReason": {
            "unavailableInRegion": "Unavailable in your region",
            "unsupportedTokenAndNetwork": "Unsupported token and network",
            "amountBelowMinimum": "Amount below the minimum",
            "unsupportedOnDevice": "Unsupported on this device",
            "amountAboveMaximum": "Amount above the maximum",
            "unavailable": "Unavailable",
            "complianceRestriction": "Compliance eligibility restriction"
        }
    },
```